TACC/Core-CMS-Custom
# Task/WP-157: View Submitter Users (New Admin Page)

## Overview

APCD admins should be able to view the records of all submitter users including basic information (names, emails, etc.) and payor codes and be able to update names and emails if needed.

## Related

- [WP-157](https://tacc-main.atlassian.net/browse/WP-157)

## Changes

The Admin dropdown in the navigation bar now contains the View Submitter Users page, which displays a table containing submitter ID, user ID, user number, user email, user name, payor code, and an Actions column of dropdown menu. These dropdown menus allow an admin to view and edit records for that specific submitter user. Only user name and user email can be edited.

## Testing

1. Click on Admin, then View Submitter Users. The table of submitter users should load.
2. Click on a user's Select Action dropdown and select View Record.
3. A modal should appear displaying the submitter user's information. Click the X to close it.
4. Click on a user's Select Action dropdown and select Edit Record.
5. Edit the user's name and/or email and click Submit. The user's information should be updated.

## UI

![image](https://github.com/user-attachments/assets/a15df148-ab2e-4076-844f-e95f9a5c21d5)
